eva-lib/src/test/java/uk/ac/ebi/eva/lib/repository/VariantEntityRepositoryImplTest.java

```diff
@@ -1,172 +0,0 @@
-package uk.ac.ebi.eva.lib.repository;
-
-import org.junit.Before;
-import org.junit.Test;
-import org.junit.runner.RunWith;
-import org.springframework.beans.factory.annotation.Autowired;
-import org.springframework.context.ApplicationContext;
-import org.springframework.context.annotation.ComponentScan;
-import org.springframework.data.mongodb.MongoDbFactory;
-import org.springframework.data.mongodb.core.convert.MappingMongoConverter;
-import org.springframework.data.mongodb.core.query.Criteria;
-import org.springframework.data.mongodb.core.query.Query;
-import org.springframework.data.mongodb.repository.config.EnableMongoRepositories;
-import org.springframework.test.context.junit4.SpringJUnit4ClassRunner;
-
-import java.util.ArrayList;
-import java.util.List;
-
-import static org.junit.Assert.assertEquals;
-
-@RunWith(SpringJUnit4ClassRunner.class)
-@EnableMongoRepositories
-@ComponentScan("uk.ac.ebi.eva.lib.repository")
-public class VariantEntityRepositoryImplTest {
```

Comment:
There's tests for filters in here: https://github.com/tomdcsmith/eva-ws/blob/filter_refactor/eva-lib/src/test/java/uk/ac/ebi/eva/lib/repository/VariantEntityRepositoryTest.java